docs/source/robust.rst

```diff
@@ -0,0 +1,20 @@
+Robust Statistics
+==========
+
+.. function:: trim(x; prop=0.0, count=0)
+
+  Return a copy of ``x`` with either ``count`` or proportion ``prop`` of the highest
+  and lowest elements removed.  To compute the trimmed mean of ``x`` use
+  ``mean(trim(x))``; to compute the variance use ``trimvar(x)`` (see [``trimvar``](@ref)).
```

Comment:
AFAIK  `@ref` won't work in RST files (re-sigh). Just drop the link for now, once we move to Documenter the docstring will be extracted automatically with its link.